01-shapefile-attributes.Rmd

```diff
@@ -294,71 +284,78 @@ levels(lines_HARV$TYPE)
 summary(lines_HARV$TYPE)
 
 #plot the lines data, apply a diff color to each category
-plot(lines_HARV, col=lines_HARV$TYPE,
+plot(lines_HARV, 
+     col=lines_HARV$TYPE,
```

Comment:
I've added palette(default).  However, how that the colors are coming out unique I notice that the color and line width don't seem to match as they should.